src/utils/index.js

```diff
@@ -1,17 +1,59 @@
-function getTodayDate() {
-  const date = new Date();
-  const days = ['일요일', '월요일', '화요일', '수요일', '목요일', '금요일', '토요일'];
-
-  return `${date.getFullYear()}. ${`${date.getMonth() + 1}`.padStart(
-    2,
-    '0',
-  )}. ${`${date.getDate()}`.padStart(2, '0')}. ${days[date.getDay()]}`;
-}
-
-function shufflePressOrder() {
-  const array = Array.from({ length: 96 }, (v, idx) => idx);
+const getTodayDate = () => {
+  const today = new Date();
+  const options = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+    weekday: 'long',
+  };
+
+  return today.toLocaleDateString('ko-KR', options);
+};
+
+const shufflePressOrder = () => {
+  const array = Array.from({ length: 96 }, (_, idx) => idx);
   array.sort(() => Math.random() - 0.5);
   return array;
-}
+};
+
+export const customQuerySelector = (selector, $target = document) => {
+  const childNodes = [...$target.childNodes];
+  const targetSelctor = selector.trim();
+  const tagType = targetSelctor.charAt(0);
+  const name = targetSelctor.slice(1);
+
+  while (childNodes.length > 0) {
+    const $node = childNodes.shift();
+
+    if (tagType === '#' && $node.id === name) return $node;
+    else if (tagType === '.' && $node.classList && $node.classList.contains(name)) return $node;
+    else if ($node.tagName === targetSelctor.toUpperCase()) return $node;
+    else {
+      const $result = customQuerySelector(selector, $node);
+      if ($result) return $result;
+    }
```

Comment:
```suggestion
    if (tagType === '#' && $node.id === name) return $node;
    
    if (tagType === '.' && $node.classList && $node.classList.contains(name)) return $node;
    
    if ($node.tagName === targetSelctor.toUpperCase()) return $node;
    
    const $result = customQuerySelector(selector, $node);
    if ($result) return $result;
```

`return` 사용하는거면 이렇게 조건문을 없애는게 어떨까요 :)

Reply:
`return`으로 함수가 종료되는데 굳이 불필요한 분기를 할 필요가 없어 보이네요!! 예리하십니다 😎